js/content.js

```diff
@@ -243,7 +243,13 @@ function removeBlur() {
 
 // timer for fading in and out (works with both)
 // changing it changes the amount of time it takes to fade out (in ms)
-let timerForFading = 50; 
+const TIMER_FADE_LOOP = 50; 
```

Comment:
Can this be prefixed with FADE_ instead of TIMER_ to match the rest of the constants?